diff --git a/lib/usb/Makefile b/lib/usb/Makefile
index 14471753..f5282bc1 100644
--- a/lib/usb/Makefile
+++ b/lib/usb/Makefile
@@ -24,11 +24,10 @@ all: libusb.a
 
 include $(CIRCLEHOME)/Rules.mk
 
-OBJS	= lan7800.o smsc951x.o usbbluetooth.o usbcdcethernet.o usbfloppydevice.o \
+OBJS	= lan7800.o smsc951x.o usbbluetooth.o usbcdcethernet.o \
 	  usbconfigparser.o usbdevice.o usbdevicefactory.o usbendpoint.o usbfunction.o \
-	  usbgamepad.o usbgamepadps3.o usbgamepadps4.o usbgamepadstandard.o usbgamepadswitchpro.o \
-	  usbgamepadxbox360.o usbgamepadxboxone.o usbhiddevice.o usbhostcontroller.o \
-	  usbkeyboard.o usbmassdevice.o usbmidi.o usbmidihost.o usbmouse.o usbprinter.o usbrequest.o \
+	  usbhiddevice.o usbhostcontroller.o \
+	  usbkeyboard.o usbmassdevice.o usbmidi.o usbmidihost.o usbrequest.o \
 	  usbstandardhub.o usbstring.o usbserial.o usbserialhost.o usbserialch341.o usbserialcp210x.o \
 	  usbserialpl2303.o usbserialft231x.o usbserialcdc.o usbtouchscreen.o dwhciregister.o
 
